content/tutorials/webrtc/infrastructure/en/index.html

```diff
@@ -221,66 +212,59 @@ <h3 id="coding-webrtc-for-signaling">Coding WebRTC for signaling</h3>
   }
 };</pre>
 
-<p>To see the offer/answer and candidate exchange processes in action, take a look at the console log for the 'single-page' video chat example at <a href="http://simpl.info/rtcpeerconnection/">simpl.info/pc</a>. If you want more, download a complete dump of WebRTC signaling and stats from the chrome://webrtc-internals page in Chrome or the opera://webrtc-internals page in Opera.</p>
+<p>To see the offer/answer and candidate-exchange processes in action, see <a href="http://simpl.info/rtcpeerconnection/">simpl.info RTCPeerConnection</a> and look at the console log for a single-page video chat example. If you want more, download a complete dump of WebRTC signaling and stats from the chrome://webrtc-internals page in Google Chrome or the opera://webrtc-internals page in Opera.</p>
 
 <h3 id="peer-discovery">Peer discovery</h3>
 
-<p>This is a fancy way of saying &mdash; how do I find someone to talk to?</p>
+<p>This is a fancy way of asking, "How do I find someone to talk to?"</p>
 
-<p>For telephone calls we have telephone numbers and directories. For online video chat and messaging, we need identity and presence management systems, and a means for users to initiate sessions. WebRTC apps need a way for clients to signal to each other that they want to start or join a call.</p>
+<p>For telephone calls, you have telephone numbers and directories. For online video chat and messaging, you need identity-and-presence management systems, and a means for users to initiate sessions. WebRTC apps need a way for clients to signal to each other that they want to start or join a call.</p>
```

Comment:
Remove hyphens from` identity-and-presence management`.

(This refers to two types of system: identity management systems and presence management systems.)

Reply:
Will fix

Reply:
Will fix